packages/gatsby-theme-docs/src/layouts/internals/layout-page-navigation.js

```diff
@@ -174,6 +177,7 @@ const LayoutPageNavigation = (props) => {
 
   const navigationContainer = (
     <nav
+      key="search-container"
```

Comment:
`key="search-container"` is used twice, here and on line 160

```suggestion
      key="navigation-container"
```

Reply:
copypasta...  done